config/exec.spc

```diff
@@ -3,13 +3,11 @@
 connection "exec_local" {
   plugin = "exec"
 
-  # Working directory to use when running commands.
+  # (local only) Working directory to use when running commands.
```

Comment:
What happens if a user sets these on a remote connection? Does it error or just do nothing?

Also, in remote connections, what directory do we land in?

Reply:
- directory param is ignored on remote connection
- in remote we land in users' home directory